Userland/Libraries/LibGUI/ComboBox.h

```diff
@@ -43,10 +43,13 @@ class ComboBox : public Frame {
     int model_column() const;
     void set_model_column(int);
 
+    void set_max_rows(size_t rows);
+    size_t max_rows() const { return m_max_rows; };
+
     void set_editor_placeholder(StringView placeholder);
     const String& editor_placeholder() const;
 
-    Function<void(const String&, const ModelIndex&)> on_change;
+    Function<void(String const&, ModelIndex const&)> on_change;
```

Comment:
Still not fixed.

Reply:
but this causes conflict

Reply:
The original commit that introduced on_change should have east const. Otherwise it wouldn't pass clang-format anymore. If you were the one to introduce on_change here (hard to tell), please move this change into that introduction commit, so that it's east const from the start. If you weren't the one, then the formatter failed somehow and you should create a separate commit that notes this. Either way, it's unrelated to the commit's changes and shouldn't be here.

Reply:
done